AlgorithmFactory/Loader.cs

```diff
@@ -158,8 +158,8 @@ private bool TryCreatePythonAlgorithm(string assemblyPath, out IAlgorithm algori
             // Set the python path for loading python algorithms.
             var pythonPath = new[]
             {
-                new DirectoryInfo(Environment.CurrentDirectory).FullName,
                 pythonFile.Directory.FullName,
+                new DirectoryInfo(Environment.CurrentDirectory).FullName,
```

Comment:
Just out of curiosity, why change the ordering here?

I know *semantically* why, so that the python directory gets preference over the current directory, but why was the reason behind this? Were we incorrectly resolving a type or some other bug raised/observed?

Reply:
Locally, VS copies the python scripts to `bin/Debug` (which is `Environment.CurrentDirectory`). When we modify the code at `Algorithm.Python` and build/rebuild, the modified code was not being copied over (I had to manually delete the old file). I tried to modify the properties to "Copy Always" or "Copy if newer", but it just didn't replace the file.

Reply:
Does this not conflict with [https://docs.python.org/3/tutorial/modules.html#the-module-search-path](url)?